modules/references/client/components/create-reference/Experience.js

```diff
@@ -17,69 +18,80 @@ export default function Recommend({
-    met: t('Would you recommend others to meet them?'),
+    hostedMe: t(
+      `Did you feel comfortable being hosted by ${referencedUserName}?`,
+    ),
+    hostedThem: t(`Did you feel comfortable hosting ${referencedUserName}?`),
+    met: t(`Did you feel comfortable meeting ${referencedUserName}?`),
   };
 
   const question = recommendQuestions[primaryInteraction];
 
   return (
     <div className="panel panel-default">
       <div className="panel-heading">
-        <h4 id="would-you-recommend-them-question">{question}</h4>
+        <h4 id="did-you-feel-comfortable-question">{question}</h4>
       </div>
       <div className="panel-body">
         <ToggleButtonGroup
           type="radio"
-          name="recommend"
+          name="comfortable"
           onChange={onChangeRecommend}
-          value={recommend}
-          aria-labelledby="would-you-recommend-them-question"
+          value={comfortable}
+          aria-labelledby="did-you-feel-comfortable-question"
         >
           <ToggleButton
-            className="btn btn-lg"
-            aria-checked={recommend === 'yes'}
+            className="btn btn-lg comfortable-reply-button"
+            aria-checked={comfortable === 'yes'}
             value="yes"
-            bsStyle="success"
+            bsStyle="default"
             bsSize="large"
           >
             {t('Yes')}
           </ToggleButton>
           <ToggleButton
-            className="btn btn-lg"
-            aria-checked={recommend === 'no'}
+            className="btn btn-lg comfortable-reply-button"
+            aria-checked={comfortable === 'no'}
             value="no"
-            bsStyle="danger"
+            bsStyle="default"
             bsSize="large"
           >
             {t('No')}
           </ToggleButton>
           <ToggleButton
-            aria-checked={recommend === 'unknown'}
+            className="btn btn-lg comfortable-reply-button"
+            aria-checked={comfortable === 'unknown'}
             value="unknown"
             bsStyle="default"
             bsSize="large"
           >
             {t("I don't know")}
           </ToggleButton>
         </ToggleButtonGroup>
-        {recommend === 'no' && (
+
+        {comfortable === 'no' && (
           <Report
             onChangeReport={onChangeReport}
             onChangeReportMessage={onChangeReportMessage}
             report={report}
             reportMessage={reportMessage}
           />
         )}
+        <span className="help-block">
+          {
+            "This answer won't be shared with them and is collected for safety reasons and aggregated analysis only. "
+          }
+          {/* <a>Read more</a> */}
+        </span>
       </div>
     </div>
   );
 }
 
-Recommend.propTypes = {
+Experience.propTypes = {
   primaryInteraction: PropTypes.string.isRequired,
-  recommend: PropTypes.string,
+  referencedUserName: PropTypes.string.isRequired,
+  comfortable: PropTypes.string,
```

Comment:
I think this prop maps to something in backend and in database model, so renaming only the clientside prop will lead to confusion.

Reply:
Would be nice to find a good word for the field, it's still `recommend` in the backend and other places. I don't have any good suggestions.

I think we could just keep it as `recommend` for now though, but if we an come up with a good alternative, it needs to be changed _everywhere_ (e.g. just below too `onChangeRecommend`).

Reply:
Agree in preference to keep it consistent for now. :-)